packages/expo-calendar/src/Calendar.ts

```diff
@@ -377,7 +377,7 @@ export async function deleteAttendeeAsync(id: string): Promise<void> {
 } // Android
 
 export async function getRemindersAsync(
-  calendarIds: string[],
+  calendarIds: (string | null)[],
```

Comment:
Interesting way, but convention is to use `Array<firstTypeInUnionType | secondTypeInUnionType | ... and so on>`